schema.graphql

```diff
@@ -482,7 +497,7 @@ type Mutation {
   cancelMembershipRequest(membershipRequestId: ID!): MembershipRequest!
   checkIn(data: CheckInInput!): CheckIn!
   createAdmin(data: UserAndOrganizationInput!): User!
-  createAdvertisement(endDate: Date!, link: String!, name: String!, orgId: ID!, startDate: Date!, type: String!): Advertisement!
+  createAdvertisement(input: CreateAdvertisementInput!): CreateAdvertisementPayload
```

Comment:
No need to return `CreateAdvertismentPayload`. We can return `Advertisement` directly.

Reply:
@EshaanAgg mutations can return more than one type of object, it's not always the case that they mutate a single object and return the same thing, sometimes they can mutate many different types of objects and those different types of mutated objects need to be returned to the client

though not a strict requirement I think they help achieve consistency in the schema's design